doc/api/inspector.md

```diff
@@ -85,7 +85,7 @@ execution is suspended (through breakpoints, for example):
 session.on('Debugger.paused', ({ params }) => {
   console.log(params.hitBreakpoints);
 });
-// [ '/node/test/inspector/test-bindings.js:11:0' ]
+// [ '/test/parallel/test-inspector-bindings.js:88:0' ]
```

Comment:
Maybe we can just use a fake name here? Something like `// [ '/the/file/that/has/the/breakpoint.js:11:0' ]` ?